issue_metrics.py

```diff
@@ -21,12 +21,21 @@
 from datetime import datetime, timedelta
 from os.path import dirname, join
 from urllib.parse import urlparse
+from typing import List
 
 import github3
 from dotenv import load_dotenv
 
 
-def search_issues(repository_url, search_query, github_connection):
+# IssueWithMetrics is a type alias for a GitHub issue with metrics attached.
+IssueWithMetrics = github3.issues.Issue
```

Comment:
Python is seeing this as a duplicate declaration of IssueWithMetric on line 357.

Reply:
Wait, ill push a commit to fix this

Reply:
```suggestion
from github3.issues.issue import Issue as IssueWithMetrics
```

Does this look good? I am directly renaming the github3.Issue type in the import statement itslf
I think this will fix the issue that python is complaining about...

Reply:
I don't think it's accurate to say that `IssueWithMetrics` is of type github3.issues.issue. I've moved the definition of `issueWithMetrics` earlier in the file so it can be referenced earlier. That seems to have resolved the issue.